setup.py

```diff
@@ -4,49 +4,49 @@
 
 here = os.path.abspath(os.path.dirname(__file__))
 
-with open(os.path.join(here, 'README.md'), 'r', encoding='utf-8') as ffile:
+with open(os.path.join(here, "README.md"), "r", encoding="utf-8") as ffile:
     README = ffile.read()
 
 setup(
-    name='django-walletpass',
-    python_requires='>=3.5.0',
-    version='2.0',
-    author='Develatio Technologies S.L.',
-    author_email='contacto@develat.io',
+    name="django-walletpass",
+    python_requires=">=3.6.0",
```

Comment:
You should add this to the CHANGELOG as well. If we're increasing the min py version we should specify it.

Reply:
done